### ISSUE TITLE
request: schema or API docs

### ISSUE THREAD

**Issue by ThinkAndWander** [MehVahdJukaar/polytone#115]
The wiki is useful, but it doesn't describe the structure of what's available comprehensively. It describes it in English and the sample pack is helpful, but what would be golden is an API doc that names all supported properties, their expected value types and variant syntax. I'm not 100% about what's available and it's not an easy resource to use in its current form. But thank you so much for the existing write-up! The English explanation is also helpful :)

**Comment by MehVahdJukaar** [comment]
they would be yeah. I have to do some research into ways of generating those before i can add them.
Given the schemas are nice for developers but almsot incomprehensible for the average joe and from doing this mod i discovered that even if you document all in plain english with examples it will still be quite hard for many, even when its very similar to existing systems like datapacks
